src/test/resources/alma-fix/990210950050206441.json

```diff
@@ -0,0 +1,240 @@
+        "type" : [ "SoftwareApplication" ],
+        "label" : "Software lobid-resources"
+      },
+      "object" : {
+        "id" : "https://lobid.org/hbz01/990210950050206441",
+        "type" : [ "DataFeedItem" ],
+        "label" : "hbz-Ressource 990210950050206441 im Exportformat MARC21 XML",
+        "inDataset" : {
+          "id" : "https://datahub.io/dataset/hbz_unioncatalog",
+          "label" : "hbz_unioncatalog"
+        }
+      }
+    },
+    "license" : [ {
+      "id" : "http://creativecommons.org/publicdomain/zero/1.0",
+      "label" : "Creative Commons-Lizenz CC0 1.0 Universal"
+    } ],
+    "sourceOrganization" : {
+      "id" : "http://lobid.org/organisations/DE-61#!",
+      "label" : "lobid Organisation"
+    },
+    "provider" : {
+      "id" : "http://lobid.org/organisations/DE-DNB#!",
+      "label" : "lobid Organisation"
+    },
+    "modifiedBy" : {
+      "id" : "http://lobid.org/organisations/DE-61#!",
+      "label" : "lobid Organisation"
+    }
+  },
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-HT018924091",
+    "label" : "Culturegraph Ressource"
+  }, {
+    "id" : "http://worldcat.org/oclc/1107003384",
+    "label" : "OCLC Ressource"
+  } ],
+  "inCollection" : [ {
+    "id" : "http://lobid.org/resources/HT014176012#!",
+    "label" : "Nordrhein-Westfälische Bibliographie (NWBib)",
+    "type" : [ "Collection" ]
+  } ],
+  "language" : [ {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/ger",
+    "label" : "Deutsch"
+  } ],
+  "extent" : "108 ungezählte Seiten : 54 Illustrationen",
+  "natureOfContent" : [ {
+    "label" : "Bildband",
+    "id" : "http://d-nb.info/gnd/4145395-5"
+  } ],
+  "subject" : [ {
+    "type" : [ "Keyword" ],
+    "source" : {
+      "label" : "Freie Verschlagwortung",
+      "id" : "https://www.wikidata.org/wiki/Q47524318"
+    },
+    "label" : "Der Ferd"
+  }, {
+    "type" : [ "Keyword" ],
+    "source" : {
+      "label" : "Freie Verschlagwortung",
+      "id" : "https://www.wikidata.org/wiki/Q47524318"
+    },
+    "label" : "Humor"
+  }, {
+    "notation" : "700",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "DNB (Systematik der Deutschen Nationalbibliographie)",
+      "id" : "http://bartoc.org/en/node/18497"
+    }
+  }, {
+    "id" : "https://nwbib.de/subjects#N841070",
+    "label" : "Künstler",
+    "notation" : "841070",
+    "type" : [ "Concept" ],
+    "source" : {
+      "id" : "https://nwbib.de/subjects",
+      "label" : "Sachsystematik der Nordrhein-Westfälischen Bibliographie"
+    }
+  }, {
+    "type" : [ "ComplexSubject" ],
+    "label" : "Ferd | Cartoon",
+    "componentList" : [ {
+      "type" : [ "Person" ],
+      "label" : "Ferd",
+      "source" : {
+        "label" : "Gemeinsame Normdatei (GND)",
+        "id" : "https://d-nb.info/gnd/7749153-1"
+      },
+      "id" : "http://d-nb.info/gnd/1093886471",
+      "gndIdentifier" : "1093886471",
+      "dateOfBirthAndDeath" : "ca. 20.-21. Jh."
```

Comment:
already covered?

Reply:
This filters to ca. and Jh. different case as the other.